features/mock/cantabular_client.go

```diff
@@ -175,3 +179,31 @@ func (c *CantabularClient) GetCategorisations(_ context.Context, _ cantabular.Ge
 	}
 	return c.GetCategorisationsResponse, nil
 }
+
+func (c *CantabularClient) GetBaseVariable(_ context.Context, _ cantabular.GetBaseVariableRequest) (*cantabular.GetBaseVariableResponse, error) {
+	if !c.Healthy {
+		return nil, dperrors.New(
+			errors.New("test error response"),
```

Comment:
found this in other places in the file too, so not a must fix, but seems the error message is not useful at all